From 8b2dc6377637c6c146ab5d23a4e119e3d508156c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?H=C3=A5kon=20=C3=98ye=20Amundsen?=
 <haakon.amundsen@nordicsemi.no>
Date: Tue, 19 Jan 2021 15:16:55 +0100
Subject: [PATCH] Move misplaced 'else' within #ifdef to avoid compilation
 error
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

... when MBEDTLS_MD_C is not defined.

Signed-off-by: Håkon Øye Amundsen <haakon.amundsen@nordicsemi.no>
---
 library/psa_crypto.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/library/psa_crypto.c b/library/psa_crypto.c
index bffddc995..8b1201461 100644
--- a/library/psa_crypto.c
+++ b/library/psa_crypto.c
@@ -5220,8 +5220,8 @@ static psa_status_t psa_key_derivation_setup_kdf(
         operation->capacity = 255 * hash_size;
         return( PSA_SUCCESS );
     }
-#endif /* MBEDTLS_MD_C */
     else
+#endif /* MBEDTLS_MD_C */
         return( PSA_ERROR_NOT_SUPPORTED );
 }
 
-- 
2.30.0

